server/reports/expiring-items/2_3_0/src/style.css

```diff
@@ -22,11 +22,31 @@
   text-align: left;
   margin-top: 10;
   margin-bottom: 15;
+  border-collapse: collapse;
 }
 
 .container table td {
-  padding: 5px;
+  padding: 10px;
   vertical-align: top;
+  border-bottom: 1px solid rgb(164, 163, 163);
+}
+
+.container table tr:last-child td {
+  border-bottom: none;
+}
+
+.container > table > thead {
+  position: -webkit-sticky;
+  position: sticky;
+  top: 0;
+  background-color: #f1f1f1;
```

Comment:
![Screenshot 2024-10-11 at 1 09 52 PM](https://github.com/user-attachments/assets/d80bc4a0-7fed-43ca-8a32-65e1f190af7d)

Wonder if the header needs a border, looks a bit funny when I print now?